lib/apply.js

```diff
@@ -30,9 +53,16 @@ function apply() {
     const arrStyle = arguments[0].split(" ")
 
     arrStyle.map((item) => {
-      obj = {
-        ...obj,
-        ...map[item]
+      if (item.includes("/")) {
+        obj = {
+          ...obj,
+          ...detectResponsive(item.split("/"))
+        }
+      } else {
+        obj = {
+          ...obj,
+          ...map[item]
+        }
```

Comment:
If it shows single `if statement` please consider using ternary operator to make it short.